mantidproject/mantid
# LoadILLTOF: add monitor spectra to the end of the workspace

This PR modifies [`LoadILLTOF`](http://docs.mantidproject.org/nightly/algorithms/LoadILLTOF-v2.html) so that the monitor data is at the end of the loaded workspace, not at the beginning. The change makes the spectrum numbers match with the detector IDs, as requested by the instrument scientists at ILL.

**To test:**

Load IN4, IN5 and IN6 data. Some test data is available under `ExternalData/Testing/Data/U
nitTest/ILL/` in Mantid's build directory. Check that

- Monitor data is at the end of the workspaces.
- Spectrum numbers and detector IDs match except for monitors, which have IDs such as 0, 1001, 1002 and 1003 (instrument dependent)

Fixes #18842.


*Release notes were updated accordingly.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
